mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/ProfileScreen.kt

```diff
@@ -66,7 +71,7 @@ fun ProfileHeader(state: ProfileScreenState, modifier: Modifier = Modifier) {
     ProfilePicture(state)
     Column(horizontalAlignment = Alignment.CenterHorizontally) {
       Text(state.name, style = MaterialTheme.typography.h5)
-      Text(state.email, style = MaterialTheme.typography.subtitle2)
+      Text("", style = MaterialTheme.typography.subtitle2)

```

Comment:
This seems a bit off to me. Why is this `Text` composable always empty ?

Reply:
Fixed in e2d18a79d8d6c16fbd877e58ae775b417ef6e1cb